### ISSUE TITLE
sometimes fadeaway for div popup doesn't happen and it just stays there

### ISSUE THREAD

**Issue by amyxzhang** [haystack/eyebrowse-chrome-ext#11]
no idea why. i've noticed this on occasion on pages like netflix and other places.

**Comment by joshblum** [comment]
hmm do you know any steps to reproduce? Does it have to do with when the computer wakes from sleep?

**Comment by amyxzhang** [comment]
i don't think so. I've noticed this definitely while watching shows on netflix. and it's happened other times but not in a reliable way (so I'll refresh and the popup won't stay the 2nd time around).

**Comment by joshblum** [comment]
ok if you notice it again check the background, popup, and current page for any errors since this will help debugging a lot. hopefully when analytics are in we can log errors back to the server and figure out issues more quickly
